src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -89,6 +89,11 @@ object ScalafixPlugin extends AutoPlugin {
         "Optional location to .scalafix.conf file to specify which scalafix rules should run. " +
           "Defaults to the build base directory if a .scalafix.conf file exists."
       )
+    val scalafixConfigOnCompile: SettingKey[Option[File]] =
```

Comment:
reading that out loud, I am wondering if this wouldn't be better
```suggestion
    val scalafixOnCompileConfig: SettingKey[Option[File]] =
```
WDYT?

Reply:
I agree 👍